src/ce.js

```diff
@@ -40,8 +40,9 @@ var ce = {
             'tag_name': elem.tagName
         };
 
-        if (_.includes(['input', 'select', 'textarea'], elem.tagName.toLowerCase())) {
-            props['value'] = this._getFormFieldValue(elem);
+        var form_field_value = this._getFormFieldValue(elem);
+        if (_.includes(['input', 'select', 'textarea'], elem.tagName.toLowerCase()) && form_field_value !== '[stripped]') {
```

Comment:
I think that instead of processing these fields twice (marking them as `stripped` and then filtering fields marked as stripped), we should just re-work the logic so that it just ignore these fields completely.